### ISSUE TITLE
You are wrongly disconnected from the mobile app, if you delete a note that was edited in the browser's extension but not synced in the mobile app

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/notes#1137]
**[Affected Platforms]:**
- Android 8.0.0 (Google Pixel)
- Android 8.0.0 (Samsung S8)
- Android 6.0 (Huawei P8 Lite)

**[Prerequisites]:**
- Have a new Firefox profile with the latest version of the "Firefox Notes" add-on (4.1.0dev custom built on 07.06.2018) installed.
- Have the latest version of the "Firefox Notes" application (1.0android-c2359) installed from "Google Play".
- You are logged into the browser's instance of "Firefox Notes".

**[Steps to reproduce]:**
1. Open the Notes application and log in if needed.
2. Open the browser with the profile from prerequisites.
3. Open any note and edit its content.
4. Go to the mobile app and select the previously edited note.
5. Press the delete button from the top right part of the screen.
6. Observe the message displayed in the bottom part of the drawer.

**[Expected result]:**
- The "Last synced at [Hour]" message is displayed.

**[Actual result]:**
- The "Reconnect to Sync" message is displayed.

**[Notes]:**
- The issue is not reproducible if a sync action is performed before deleting the note.
- Attached a screen recording of the issue:
![rec of the issue](https://user-images.githubusercontent.com/31243259/41099613-b84f21d4-6a67-11e8-8cf4-7048d1faa31f.gif)

**Comment by vladikoff** [comment]
I cannot reproduce this, could you rephrase the STR , hmmm

**Comment by vladikoff** [comment]
is the core of the issue that you edit something in the webextension and then the change cause problems in the mobile apps?

**Comment by vladikoff** [comment]
This should be fixed if you run latest extension 4.2.0 rc2 and latest mobile app

**Comment by Softvision-MariusComan** [comment]
This issue is no longer reproducible with the latest version of the "Firefox Notes" add-on (4.2.0 RC2) and with the latest version of the "Firefox Notes" application (notes-1_0-qa-v2516 custom built on 25.06.2018) installed.
Verified on Windows 10 x64, Mac 10.13, Ubuntu 14.04 x64 and on Samsung S8 (Android 8.0.0), Huawei P8 Lite (Android 6.0), Pixel (Android 8.0.0).
